documentation_crawl4ai/adaptive-strategies.md

```
@@ -0,0 +1,520 @@
+[Crawl4AI Documentation (v0.7.x)](https://docs.crawl4ai.com/)
+  * [ Home ](https://docs.crawl4ai.com/)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)